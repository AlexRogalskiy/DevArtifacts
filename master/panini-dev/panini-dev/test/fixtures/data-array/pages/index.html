{{#each breakfast.items}}
{{this}}
{{/each}}
{{#each lunch.items}}
{{this}}
{{/each}}
