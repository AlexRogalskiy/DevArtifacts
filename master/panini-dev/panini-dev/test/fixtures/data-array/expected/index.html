<html>
  <body>
    eggs
    bacon
    toast
    ham
    cheese
    chips
    
  </body>
</html>
